src/cli/rustup_mode.rs

```diff
@@ -130,10 +134,8 @@ pub fn cli() -> App<'static, 'static> {
                 .long("quiet"),
         )
         .arg(
-            Arg::with_name("toolchain")
-                .help(TOOLCHAIN_ARG_HELP)
-                .long("toolchain")
-                .takes_value(true),
+            Arg::with_name("keyword")
```

Comment:
I wonder if it would be helpful for the user experience if we use `+keyword` here instead.

Reply:
Better would be `toolchain` rather than `keyword`